content/docs/command-reference/repro.md

```diff
@@ -93,6 +96,25 @@ up-to-date and only execute the final stage.
 
 ## Options
 
+- `<target>` - Specify the stages to reproduce.
+
+  Target can be a name of the stage in the dvc.yaml file or a path to a .dvc or
```

Comment:
here we can now put `dvc.yaml`. `.dvc`, etc into `inline code`

Reply:
@skshetry please, address this ^^

Reply:
Done.